src/modules/Library/features/Popups/AddToBuilderPopup/AddToBuilderPopup.utils.tsx

```diff
@@ -282,9 +296,41 @@ export const getPerformanceTaskType = (responseType: ItemResponseType) => {
   return performanceTaskType;
 };
 
-// TODO: Make filtering for scores, subscales, and item flows, regarding which items were selected, Activity to the
-//  Reviewer dashboard assessment checkbox (can be only one in all activities), names for ABTrails activities to be
-//  unique
+const getUniqueActivityName = <T extends { name: string }>(
```

Comment:
it seems the same function is in builder, pls avoid code duplication

Reply:
done